src/client.ts

```diff
@@ -117,6 +117,21 @@ export class Client {
     }
   }
 
+  /**
+   * Queries the receipt corresponding to a transaction hash
+   *
+   * @param Transaction hash returned by call transaction
+   * @return EvmTxReceipt protobuf
+   */
+  async txReceiptAsync(txHash: Uint8Array): Promise<Uint8Array | void> {
```

Comment:
As I pointed out in @eduardonunesp PR which I guess is based on your branch `txReceiptAsync` should be renamed to `getTxReceiptAsync`, and it should explicitly return `null` if there is no result.